CHANGELOG.md

```diff
@@ -8,9 +8,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 ## 0.2.0 - 2017-10-03
 ### Added
 - Deploy to Deployment Slots
-- Local Git deployment
```

Comment:
Should this go in the "Unreleased" section above?

Reply:
Good point.